java/fury-core/src/main/java/org/apache/fury/logging/FuryLogger.java

```diff
@@ -23,88 +23,139 @@
 import java.time.format.DateTimeFormatter;
 
 public class FuryLogger implements Logger {
+
+  private static final int ERROR_LEVEL = 0;
+
+  private static final int WARN_LEVEL = 1;
+
+  private static final int INFO_LEVEL = 2;
+
+  // Default is INFO level.
+  private static final int LEVEL = INFO_LEVEL;
```

Comment:
how aboubt make it as a final field of `FuryLogger`, and pass it from `LoggerFactory` to allow users to custimize level

Reply:
Pass in `Level` when calling `LoggerFactory#getLogger`, and then pass in `Level` through the `FuryLogger` constructor?

Reply:
We can add a static method in LoggerFactory named `setLoggerLevel`, when creating logger, we can pass that level to FuryLogger constructor